ios/Exponent/Kernel/Services/EXDeviceInstallationUUIDService.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#import <Foundation/Foundation.h>
+#import <ABI39_0_0ExpoKit/ABI39_0_0EXConstantsBinding.h>
+#import <ABI38_0_0ExpoKit/ABI38_0_0EXConstantsBinding.h>
+#import <ABI37_0_0ExpoKit/ABI37_0_0EXConstantsBinding.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+// A kernel service allowing versioned expo-constants to access device installation UUID
+__deprecated_msg("The installation ID API is deprecated and will be removed with SDK39")
```

Comment:
This kernel service will stop being used once we remove SDK 39 (I hope these changes make it before SDK 40 versioning and SDK 39 will be the last SDK where the kernel service will be used). However, your comment made me rethink when will we be able to remove `[EXKernel deviceInstallationUUID]` which we'll be able to remove once:
- SDK 39 is removed (since kernel service used by versioned `EXConstantsBinding` uses `EXKernel` under the hood) **and**
- legacy Notifications API is removed,

which will be further in the future than just SDK 39 removal. I'll fix the comment there to mention legacy Notifications too.

Reply:
Actually updated both `deprecated` messages with:
> The installation ID API is deprecated and will be removed once both SDK 39 and legacy Notifications API are removed

and added
```
// TODO: Remove this after SDK 39 is phased out
```
over `EXDeviceInstallationUUIDService`.